src/components/tooltip/bl-tooltip.css

```
@@ -3,34 +3,17 @@
   cursor: pointer;
 }
 
-.tooltip {
-  position: fixed;
-  box-sizing: border-box;
-  border: none;
-  background-color: var(--bl-color-secondary);
-  color: var(--bl-color-content-primary-contrast);
-  border-radius: var(--bl-size-3xs);
-  pointer-events: none;
-  font: var(--bl-font-title-3-regular);
-  padding: var(--bl-size-m);
-  z-index: 999;
-  width: max-content;
-  hyphens: auto;
+.wrapper{
+  display: inline-block;
+}
 
+.tooltip {
+  --bl-popover-background-color: var(--bl-color-secondary);
+  --bl-popover-border-color: var(--bl-color-secondary);
```

I think we should use content color instead.

```suggestion
  --bl-popover-background-color: var(--bl-color-content-primary);
  --bl-popover-border-color: var(--bl-color-content-primary);
```